examples/build.gradle

```diff
@@ -27,7 +28,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.mapbox.navigation.examples"
+        applicationId "com.mapbox.navigation.test_app"
```

Comment:
this build gradle is in examples, so why is it being changed to test_app? It's in the examples module so it should stay as examples

![Screen Shot 2021-03-29 at 5 19 16 PM](https://user-images.githubusercontent.com/3021882/112915337-e70e8900-90b2-11eb-9f47-9fa2f07937f5.png)
^ Firebase has 2 projects right now

For what it's worth.. having examples and test_app has caused a bunch of issues and is confusing. So it would be best if we picked on.

Reply:
https://app.circleci.com/settings/project/github/mapbox/mapbox-navigation-android/environment-variables

^ There is a `TEST_APP_GOOGLE_SERVICES_JSON` that needs to be changed.

Reply:
https://github.com/mapbox/mapbox-navigation-android/blob/2bcb4404e8afdc201a897e4aeb60f692112ef939/.circleci/config.yml#L285

Here's where the google-services-json is copied over for circleci

Reply:
Fine by me if you want to merge this and make this a follow up

Reply:
Interesting to see that examples in Firebase has a different package name than what we have in the SDK.

Reply:
> this build gradle is in examples, so why is it being changed to test_app?

This should probably have the same package name as it in Firebase.